### DISCUSSION TITLE
problems with putting cards on your hand

### DISCUSSION THREAD

**Discussion by DERTAZ** [bafolts/terraforming-mars#5080] [Content Request]
In the past, you could arrange the cards in your hand in any order - by catching them with the cursor and dragging them. It has not been working properly for several months, it is not possible to arrange any cards, only you can move the cards in the first row, and this can be problematic.
Has it only broken for me? is it the same for everyone?

**Comment by bafolts** [comment]
@DERTAZ which browser are you using?

**Reply by DERTAZ** [reply]
Chrome

**Reply by DERTAZ** [reply]
@bafolts I checked on most browsers it does not work properly. In the CHROME, EDGE, OPERA viewer, cards can be placed only in the first row. Does anyone have the same? How can this be fixed?

**Comment by kberg** [comment]
Which browser version, and which operating system? If this is on a phone , which phone version?

**Comment by bafolts** [comment]
Can you provide a video of the issue? I am able to order cards successfully in the 2nd row on terraforming-mars.herokuapp.com.
